### DISCUSSION TITLE
file transfer to a new drive

### DISCUSSION THREAD

**Discussion by West-Tek** [ppy/osu#22757] [Q&A] [answered]
how can i modify the route of all osu lazer songs??

**Comment by ItsShamed** [comment, marked as answer]
Duplicate of #20409? 

> If you just want to move the user data, there's a "change folder" button towards the top of settings.
> 
> ![osu! 2022-09-23 at 04 07 47](https://user-images.githubusercontent.com/191335/191890467-91a140fd-4a45-4a25-895d-41b06e815687.png)
>

**Reply by West-Tek** [reply]
ty alredy  done it but now all my maps are one the folder but idont know how to import them again

**Reply by nagi-desuuu** [reply]
Can you elaborate on that further?

**Reply by West-Tek** [reply]
I moved the osu install folder to other disk and then the folder with all the songs, but it doesn't give me the option of importing all of them like stable F5

**Reply by nagi-desuuu** [reply]
Did you try checking out `Run Setup Wizard`?

![image](https://user-images.githubusercontent.com/75596775/222094820-657c5b5a-4e45-46fe-911d-4ddb53ca028b.png)

It has an option to import your stable beatmaps from there.

![image](https://user-images.githubusercontent.com/75596775/222095100-20037bc5-85f8-4c7c-9a3d-3d48b4cb8fb5.png)

I recall this being an intentional thing that the quick import files buttons were removed on releases (making them visible on debug builds only) in favor of the setup screen, but I can't find the exact issue where it appears.

**Reply by West-Tek** [reply]
Will try it ty

**Reply by West-Tek** [reply]
But I already deleted the old folder .

**Reply by West-Tek** [reply]
didnt worked
the only thing i can do is download all maps one by one
